packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -13,7 +13,7 @@ import { ITokenClaims } from "@fluidframework/protocol-definitions";
 import { KJUR as jsrsasign } from "jsrsasign";
 import { v4 as uuid } from "uuid";
 
-export const defaultTinyliciousPort = 35843;
+export const defaultTinyliciousPort = 7070;
```

Comment:
FYI - Hopefully you won't need this soon (see PR #5630).

Reply:
Thank you, @DLehenbauer  !